### ISSUE TITLE
Unify default test docker image name across code base

### ISSUE THREAD

**Issue by grahamwhaley** [clearcontainers/tests#411]
Many of our tests and metrics use the same small docker image as the base of their tests (generally busybox at present). Currently that is hard coded across the code base. We should unify it into a single setting, probably in test-versions.txt.

Having said that, I did this and then 'source'd that into metrics/lib/common.bash, and then we run into source ../.. issues.

```
 metrics/lib/common.bash
 @@ -15,6 +15,9 @@
  # limitations under the License.
  
  SCRIPT_PATH=$(dirname $(readlink -f $0))
 +
 +source "${SCRIPT_PATH}/../../test-versions.txt"
 +
  RESULT_DIR="${SCRIPT_PATH}/../results"
  LIB_DIR="${SCRIPT_PATH}/../lib"
```

```
 test-versions.txt
 @@ -11,3 +11,8 @@ kernel_version="4.5-50"
  
  # Go version to use for building and testing Clear Containers
  go_version="1.8.3"
 +
 +# Default docker image used for tests and metrics, unless your test has a
 +# specific need to use a different image (to get specific tools etc.)
 +# Generally we keep this to be the smallest useful image we can find
 +default_docker_image="busybox"
```

At that point you can run the individual tests by hand, but running them via the `run_all_metrics.sh` causes a source file not found issue, because the `run_all_metrics.sh` file itself sources `common.bash`, from a different directory level.

@bwarden looked into this a bit, and potential solutions. I'll paste verbatim:

```
Right, the problem is that $0 is the script you actually ran. This looks like a clue:
https://unix.stackexchange.com/questions/4650/determining-path-to-sourced-shell-script

tl;dr looks like we should check ${BASH_SOURCE[0]} (and maybe use a different name than SCRIPT_PATH, since we use that within the test scripts themselves).
```
